src/common/Constants.vala.in

```diff
@@ -29,6 +29,7 @@ namespace PC.Constants {
     public const string PARENTAL_CONTROLS_ACTION_ID = "io.elementary.settings.screentime-limits.administration";
     public const string PLANK_CONF_DIR = "/.config/plank/dock1/settings";
     public const string DAEMON_CONF_FILE = "@DAEMON_CONF_DIR@/daemon.conf";
```

Comment:
Ah but maybe we should rename the default config file so that it won't conflict with the current config file for other packaging systems.